content/en/docs/security/security-response.md

```diff
@@ -7,12 +7,9 @@ Security vulnerabilities should be handled quickly and sometimes privately. The
 primary goal of this process is to reduce the total time users are vulnerable to
 publicly known exploits.
 
-The
-[OpenTelemetry Technical Committee (OTel TC)](https://github.com/open-telemetry/community/blob/main/tech-committee-charter.md)
-and relevant repository maintainers, supported by tooling provided by the
-[SIG Security](https://github.com/open-telemetry/sig-security), are responsible
-for responding to the incident organizing the entire response including internal
-communication and external disclosure.
+The relevant OTel repository maintainers, supported by the Security SIG and
```

Comment:
I think(?) the website tries to avoid the OTel abbrevation
```suggestion
The relevant OpenTelemetry repository maintainers, supported by the Security SIG and
```